cmd/bosun/sched/sched.go

```diff
@@ -607,18 +621,17 @@ func (s *State) Touch() {
 // latest status. Returns the previous status.
 func (s *State) Append(event *Event) Status {
 	last := s.Last()
-	if len(s.History) == 0 || s.Last().Status != event.Status {
-		event.Time = time.Now().UTC()
+	if len(s.History) == 0 || last.Status != event.Status {
 		s.History = append(s.History, *event)
 	}
 	return last.Status
 }
 
-func (s *State) Last() Event {
+func (s *State) Last() *Event {
```

Comment:
I can't see where we need this to be a `*Event` instead of an Event. I don't mean that there's some caller who requires a *Event, I mean I can't see where any caller needs a *Event for a good reason as opposed to a normal Event.